jobsdb/jobsdb.go

```diff
@@ -1422,8 +1432,8 @@ func (jd *HandleT) createDSInTx(tx *sql.Tx, newDS dataSetT) error {
                                      job_id BIGINT REFERENCES %q(job_id),
                                      job_state VARCHAR(64),
                                      attempt SMALLINT,
-                                     exec_time TIMESTAMP,
-                                     retry_time TIMESTAMP,
+                                     exec_time TIMESTAMP NOT NULL DEFAULT NOW(),
+                                     retry_time TIMESTAMP NOT NULL DEFAULT NOW(),
```

Comment:
Don't feel database migration files are necessary because
a. we inject exec_time & retry_time from our code.
b. these tables are transient.

Reply:
why do we have the need to remove these default values?